### ISSUE TITLE
Allow different names for the SAR application

### ISSUE THREAD

**Issue by benbridts** [awslabs/aws-deployment-framework#342]
[The documentation says:](https://github.com/awslabs/aws-deployment-framework/blob/master/docs/serverless-application-repo.md)
> Application Name
> The stack name of this application created via AWS CloudFormation. This should always be aws-deployment-framework.

However, if you want to install ADF using CloudFormation / SAM, there is no way to select the name . Instead the resulting (nested) CloudFormation stack will have a name like `${parentStack}-${resourceName}-${randomId}`. Eg. if you deploy the template below as 'adf-test', it might look like `adf-test-AwsDeploymentFrameWork-W4AUMAFJESS7`.

I looked at the resulting template, and I couldn't find any use of `AWS::StackName` in it. This leads me to a few questions:
- Is it really needed to use `aws-deployment-framework` as the name?
- If not, can we update the documentation?
- Is there anything that would block using CloudFormation to deploy the SAR app?

```yaml
Description: AWS Deployment Framework SAR app and related resources
Parameters:
  Version:
    Description: ADF Version
    Type: String
    Default: 3.1.2
Transform: AWS::Serverless-2016-10-31
Resources:
  AwsDeploymentFrameWork:
    Type: AWS::Serverless::Application
    Properties:
      Location:
        ApplicationId: arn:aws:serverlessrepo:us-east-1:112893979820:applications/aws-deployment-framework
        SemanticVersion: !Ref Version
```

**Comment by benbridts** [comment]
Update: I found 1 hardcoded values in the template that depends on this naming (searched for `serverlessrepo-aws`):

https://github.com/awslabs/aws-deployment-framework/blob/master/src/template.yml#L414

**Comment by sbkok** [comment]
Thank you for your patience. I am happy to inform you that this issue has been [resolved in our latest release v3.2.0](https://github.com/awslabs/aws-deployment-framework/releases/tag/v3.2.0) just now.
I'm hereby closing this issue. Please open a new issue if you are experiencing any issues with the latest release.
